src/js/angular/import/directives/import-resource-tree.directive.js

```diff
@@ -93,13 +95,17 @@ function importResourceTreeDirective($timeout) {
             };
 
             $scope.onResetStatus = () => {
-                if ($scope.selectedResources && $scope.selectedResources.length > 0) {
-                    $scope.onReset({selectedResources: $scope.selectedResources});
+                if (!$scope.selectedResources) {
+                    return;
+                }
+                const files = $scope.selectedResources.filter((resource) => !resource.isDirectory());
```

Comment:
I've added additional methods here for getting the files and filenames only. But you can leave this now. We can change it later